posthog/api/signup.py

```diff
@@ -122,7 +139,7 @@ def create_team(self, organization: Organization, user: User) -> Team:
 
     def to_representation(self, instance) -> Dict:
         data = UserBasicSerializer(instance=instance).data
-        data["redirect_url"] = "/ingestion" if not settings.DEMO else "/"
+        data["redirect_url"] = get_verification_redirect_url(data["uuid"])
```

Comment:
This explicitly used to send a redirect URL to `/ingestion`. But we handle this redirect in the frontend anyway if the project hans't `completed_snippet_onboarding`. So I just ignored it to make this function simpler.... but lmk if we need to consider that again.